providers/ns1/ns1Provider.go

```diff
@@ -88,6 +89,52 @@ func (n *nsone) GetZoneRecords(domain string) (models.Records, error) {
 	}
 	return found, nil
 }
+// Get DNSSEC status for zone. Returns true for enabled, false for disabled
+// a domain in NS1 can be in 3 states:
+//   1) DNSSEC is enabled  (returns true)
+//   2) DNSSEC is disabled (returns false)
+//   3) some error state   (return false plus the error)
+func (n *nsone) GetZoneDNSSEC(domain string) (bool, error) {
+	_, _, err := n.DNSSEC.Get(domain)
+
+	// rest.ErrDNSECNotEnabled is our "disabled" state
+	if err != nil && err == rest.ErrDNSECNotEnabled {
+		return false, nil
+	}
+
+	// any other errors not expected, let's surface them
+	if err != nil {
+		return false, err
+	}
+
+	// no errors returned, we assume DNSSEC is enabled
+	return true, nil
+}
```

Comment:
Add a blank line between each function unless there is a specific reason not to.